lib/index.js

```diff
@@ -236,14 +236,15 @@ const KiteAPI = {
     );
   },
 
-  requestRelatedCode(editor, filename, line, selection) {
+  requestRelatedCode(editor, filename, line, selection, editor_path) {
     return this.request(
       {
         path: '/codenav/editor/related',
         method: 'POST',
       },
       JSON.stringify({
         editor,
+        editor_path,
```

Comment:
IntelliJ is currently sending `editor_install_path` to kited. We can either change that in IntelliJ too or go with `editor_path` here. I slightly prefer the verbose `editor_install_path` but what do you think?

Reply:
Ah, thanks for letting me know. I'll change this to `editor_install_path`.